apps/web/src/common/modules/widgets/DailyUpdates.vue

```diff
@@ -430,7 +413,17 @@ export default {
     }
 }
 
-.card-wrapper .daily-update-card-alert {
+/* custom design-system component - p-data-loader */
+:deep(.p-data-loader) {
+    .data-loader-container {
+        .loader-wrapper {
+            .loader {
+                display: block;
+            }
+        }
+    }
+}
+.p-data-loader > .data-loader-container > .loader-wrapper > .loader .card-wrapper .daily-update-card-alert {
```

Comment:
question: what's the difference with line 417?

Reply:
Oh, my mistake. Thanks!!! 😂